src/test/java/org/candlepin/test/DatabaseTestFixture.java

```diff
@@ -650,6 +651,20 @@ protected Owner createOwner(String key, String name) {
         return owner;
     }
 
+    protected Owner createOwnerEnt(String key) {
```

Comment:
Let's rebase this against the changes made in the other PR so we can use the better naming convention.

Reply:
ack, done